src/Reports/SubsiteReportWrapper.php

```diff
@@ -59,9 +60,13 @@ public function columns()
     /**
      * @param arary $params
      * @return void
+     * @deprecated 2.8.0 Will be removed without equivalent functionality to replace it
      */
     public function beforeQuery($params)
```

Comment:
Doesn't this method exist in the superclass?

Reply:
Ah yup, will underprecate